index.html

```diff
@@ -251,13 +249,21 @@ <h2>Defining an RDF Dataset Hash</h2>
       The main challenge for the Working Group was to provide a standard for the <a href="#canonicalization">RDF Dataset Canonicalization function</a>. 
     </p>
 
+    <p class="note">The immediate result of the <a data-cite="RDF-CANON#canonicalization">Canonicalization Algorithm</a> is a canonical representation of the RDF dataset represented as N-Quads, using canonical blank node identifiers, and placed in code point order.
+      As the media type for N-Quads is defined in [[[n-quads]]] [[n-quads]] as `application/n-quads` no further media registration is needed for this form.
```

Comment:
```suggestion
      Since the media type for N-Quads is defined in [[[n-quads]]] [[n-quads]] as `application/n-quads`, no further media registration is needed for this form.
```